dspace/config/local.cfg.EXAMPLE

```diff
@@ -236,3 +236,11 @@ db.password = dspace
 #  LDN INBOX SETTINGS  #
 ########################
 ldn.enabled = true
+
+#---------------------------------------------------------------#
+#---------------PERSONAL DATA STORED IN PROVENANCE--------------#
+#---------------------------------------------------------------#
+# Behavior to generate provenance message, if set true,
+# personal data (e.g. email) of submitter will be hidden in dc.description.provenance
+# true - personal
+#dc.description.provenance.privacy = true
```

Comment:
This should be moved to the `dspace.cfg` where it should remain commented out.  The `local.cfg.EXAMPLE` is meant to only contain a subset of configs...primarily those that most sites will need to modify.  Therefore, this should only be added to `dspace.cfg`.

You might also consider renaming this to be `metadata.privacy.dc.description.provenance`.  That would be similar to the existing `metadata.hide.*` settings that [also appear in `dspace.cfg`](https://github.com/DSpace/DSpace/blob/main/dspace/config/dspace.cfg#L959-L975) & this setting could be moved alongside that one.  That rename would also potentially allow us to add more `metadata.privacy.*` fields to this list in the future (but for this PR, I'm fine with it _only_ allowing for dc.description.provenance)

Finally, line 245 here also could likely be removed. I don't understand what "true - personal" means.  Instead, maybe it should say "Defaults to false", to make it clear that this is disabled by default.